src/transaction/buffer.rs

```diff
@@ -275,6 +275,20 @@ impl Buffer {
             }
         }
     }
+
+    pub async fn get_write_size(&self) -> u64 {
+        let mutations = self.mutations.lock().await;
+        let mut size = 0_u64;
```

Comment:
```suggestion
        let mut size = 0;
```

And remove the casts below

Reply:
The function `calc_txn_lock_ttl` is the same as TiDB. But there is no document in TiDB too.